flang/include/flang/Parser/dump-parse-tree.h

```diff
@@ -884,8 +884,10 @@ class ParseTreeDumper {
     } else if constexpr (HasSource<T>::value) {
       return x.source.ToString();
 #endif
-    } else if constexpr (std::is_same_v<T, std::string>) {
```

Comment:
are you sure that you want to delete this case?  is it redundant or something?

Reply:
Yes, it's already covered at line 874.